resources/lang/de-DE.yml

```diff
@@ -715,6 +715,7 @@ msg_error_ta_plot_claim: 'Der Spielername fehlt. Bsp: /ta plot claim PlayerNameH
 msg_error_no_player_with_that_name: 'Es konnte kein Spiel Spieler mit dem Namen %s gefunden werden.'
 msg_admin_has_given_you_a_plot: 'Ein Admin hat für dich einen Plot bei %s beansprucht.'
 msg_town_forcepvp_setting_set_to: 'Die PvP-Einstellung der Stadt %s wurde geändert zu %s'
+msg_town_forcedisabledpvp_setting_set_to: 'The town %s has had their admindisabledpvp setting forced to %s'
```

Comment:
```suggestion
msg_town_forcedisabledpvp_setting_set_to: 'Die gezwungene PVP-Einstellung der Stadt %s wurde geändert zu %s'
```

Reply:
Translations are to be done on crowdin @nicosammito